src/main/java/Duke.java

```diff
@@ -1,10 +1,275 @@
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.io.FileNotFoundException;
+
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    private static final String FILEPATH = "ip/data/duke.txt";
+    private static final String FOLDERPATH = "ip/data";
+    private static final int TODO_NAME_CONSTANT = 5;
+    private static final int DEADLINE_NAME_CONSTANT = 9;
+    private static final int DEADLINE_BY_CONSTANT = 5;
+    private static final int EVENT_NAME_CONSTANT = 6;
+    private static final int EVENT_AT_CONSTANT = 5;
+    public static final String LINE = "____________________________________________________________";
+
+    private static ArrayList<Task> list = new ArrayList<Task>();
+
+    public static void initList() {
+        try {
+            File file = new File(FILEPATH);
+            Scanner s = new Scanner(file); // create a Scanner using the File as the source
+            while (s.hasNext()) {
+                String input = s.nextLine();
+                String[] inputWords = input.split(" / ");
+                switch(inputWords[0]) {
+                case "T":
```

Comment:
Hardcoding these letters here seems like a case of "Magic Numbers". Perhaps, you may want to consider declaring them as final constants with appropriate names near the top of the class.